hnn_core/network.py

```diff
@@ -14,6 +15,52 @@
 from .params import create_pext
 
 
+def read_spikes(fname, gid_dict=None):
+    """Read spiking activity from a collection of spike trial files.
+
+    Parameters
+    ----------
+    fname : str
+        Wildcard expression (e.g., '<pathname>/spk_*.txt') of the
+        path to the spike file(s).
+    gid_dict : dict of lists or range objects | None
+        Dictionary with keys 'evprox1', 'evdist1' etc.
+        containing the range of Cell or input IDs of different
+        cell or input types. If None, each spike file must contain
+        a 3rd column for spike type.
+
+    Returns
+    ----------
+    spikes : Spikes
+        An instance of the Spikes object.
+    """
+
+    spiketimes = []
+    spikegids = []
+    spiketypes = []
+    for file in sorted(glob(fname)):
+        spike_trial = np.loadtxt(file, dtype=str)
```

Comment:
what do you think of using a convention that singular is a single trial and plural is multiple trails?

So `spike` when it's just one trial and `spikes` when it's multiple trials.

Reply:
I'm afraid this will be ambiguous for others who haven't been a part of this conversation since the name of the class is `Spikes` (i.e, which includes spikes from either one or multiple trials).

Reply:
okay fair enough, we can think  of this later